### ISSUE TITLE
Fix data tooltips

### ISSUE THREAD

**Issue by anderspitman** [iobio/iobio-charts#70]
Production bam has small tooltips that pop up when you hover over the bars in the histograms. These are currently broken. The fact that they are broken might have been hidden for a while because we were setting the title on the custom elements, which created a single unwanted tooltip for the entire chart (see https://github.com/iobio/iobio-charts/issues/54).

**Comment by anderspitman** [comment]
One clue so far. Looks like the tooltip is showing up in the very bottom left of `document.body`

**Comment by YangQi007** [comment]
Can we close this issue? It looks like it was resolved in #54.

**Comment by anderspitman** [comment]
I think we should probably wait to close this until the formatting is properly merged

**Comment by YangQi007** [comment]
> @YangQi007 what's the status of this one? Currently the tooltips for bam1 and bam2 are different.

Yes, we need to format the tooltips in bam2. This issue is related to the  #116.
